### ISSUE TITLE
SA1642: Language ignored in .NET 6.0 SDK RC2

### ISSUE THREAD

**Issue by rsking** [DotNetAnalyzers/StyleCopAnalyzers#3397]
In .NET 6.0 SDK RC2, the language is not being taken into account when reporting SA1642

The language is setup correctly in stylecop.json

``` json
{
  "settings": {
    "documentationRules": {
      "documentationCulture": "en-GB"
    }
  }
}
```

And the constructor comment uses `Initialises` rather than `Initializes`

``` csharp
/// <summary>
/// Initialises a new instance of the <see cref="Class1"/> class.
/// </summary>
```

Expected:

No warning is issued

Actual:

SA1642 warning is issued

Remarks:

This seemed to work on preview versions of .NET 6.0 SDK, and continues to work through Visual Studio 2022.

**Comment by Daeymon** [comment]
This issue is now in the stable version of Visual Studio 2022 17.4.1.

**Comment by DEVGITKZVTH** [comment]
Also in 17.5.5

**Comment by rsking** [comment]
> Is there a workaround for this?
> 
> Just migrated from VS2019 where everything worked. Should / could I install an earlier version of StyleCopAnalyzers from NuGet to get the localisation fixed, or it a VS2022 issue and I'm stuck with the warnings?
> 
> Currently using v1.2.0.435

No work around at the moment, just suppress those warnings, which defeats the purpose of the warnings. Downgrade will not work, as the issue is with Roslyn.

**Comment by bjornhellander** [comment]
@sharwell, is there anything that could be done here? For example, are the satellite assemblies required or could the resource strings be embedded in the main assembly somehow?

**Comment by AndreR** [comment]
Seems like this bug ist still open. Are there any plans to fix it? Same problem with the German language pack.
